testsuite/libra-fuzzer/fuzz/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "libra_fuzzer_fuzz"
+name = "libra-fuzz"
```

Comment:
This commit probably should come before the one that introduces the lint (unless it does and github's ui is lying to me)

Reply:
it does, the github UI is in fact lying to you (it's because it uses author dates and not committer dates)

Reply:
I still cannot fathom why Github orders things by date rather than by, you know, the actual commit order